packages/app/src/components-styled/line-chart/index.tsx

```diff
@@ -70,6 +87,17 @@ export function LineChart<T extends Value>({
     hideTooltip,
   } = useTooltip<T & TrendValue>();
 
+  const [markerInfo, setMarkerInfo] = useState<
+    | {
+        xPosition: number;
+        yPosition: number;
+        height: number;
+        data: T;
+        margins: ChartMargins;
+      }
+    | undefined
+  >(undefined);
```

Comment:
type `undefined` can be inferred by the undefined initial value:

```suggestion
  const [markerInfo, setMarkerInfo] = useState<{
    xPosition: number;
    yPosition: number;
    height: number;
    data: T;
    margins: ChartMargins;
  }>();
```